e2e-tests/users/run

```diff
@@ -81,4 +81,13 @@ kubectl_bin apply \
 sleep 15
 compare_mysql_cmd "select-4" "SHOW TABLES;" "-h $cluster-proxysql -uoperator -p$pass"
 
+newpass="test-password2"
+newpassencrypted=$(echo -n "$newpass" | base64)
+spinup_pxc "$cluster" "$test_dir/conf/some-name.yml" 3 10
+wait_cluster_consistency "$cluster" 3 3
+patch_secret "my-cluster-secrets" "monitor" "$newpassencrypted"
+sleep 15
+wait_cluster_consistency "$cluster" 3 3
+compare_mysql_cmd "select-3" "SHOW DATABASES;" "-h $cluster-haproxy -umonitor -p$newpass"
```

Comment:
what the difference between this test and test in 44-48 lines?

Reply:
we check haproxy here

Reply:
thank you!